valkey/README.md

```
@@ -0,0 +1,222 @@
+store := valkey.New(valkey.Config{
+    InitAddress:    []string{":6379", ":6380"},
+})
+
+// Create a client with support for TLS
+cer, err := tls.LoadX509KeyPair("./client.crt", "./client.key")
+if err != nil {
+	log.Println(err)
+	return
+}
+tlsCfg := &tls.Config{
+	MinVersion:            tls.VersionTLS12,
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)